editor-open.cabal

```diff
@@ -55,20 +59,20 @@ executable editor-open-test_yaml_file
   -- other-modules:       
   -- other-extensions:    
 
--- executable editor-open-test_yaml_file_conduit
---   buildable: False
---   hs-source-dirs:      tests/
---   default-language:    Haskell2010
---   build-depends:
---      base ==4.8.*
---    , bytestring
---    , conduit >=1.2.3 && <1.3
---    , conduit-extra
---    , editor-open
---    , resourcet
---   main-is:             test_yaml_file_conduit.hs
---   -- other-modules:       
---   -- other-extensions:    
+executable editor-open-test_yaml_file_conduit
+   buildable: True
+   hs-source-dirs:      tests/
+   default-language:    Haskell2010
+   build-depends:
+      base ==4.8.*
+    , bytestring
+    , conduit >=1.2.3 && <1.3
```

Comment:
Upper bounds are evil. 

On second thought, I may have put that there, actually.

In that case, upper bounds are great.

(But remove this one)